arkouda/numeric.py

```diff
@@ -1722,6 +1725,87 @@ def where(
     return create_pdarray(type_cast(str, repMsg))
 
 
+def matching_shapes(pda_a: pdarray, pda_b: pdarray):
+    """
+    For now, this will only check that two pdarrays have matching sizes.
+    It's included so that we can put placeholders in for the check as
+    we implement n-dimensional array computation.
```

Comment:
is this a function in numpy? If not, I'm not sure how I feel about it being exposed to the user. We could make this a hidden helper by appending a `_`, so `_matching_shapes`

But I think instead of having a function for this we could just do `pda_a.shape == pda_b.shape` and cover the single and multi-dim cases